pkg/kubelet/pod/mirror_client.go

```diff
@@ -104,6 +136,17 @@ func (mc *basicMirrorClient) DeleteMirrorPod(podFullName string, uid *types.UID)
 	return true, nil
 }
 
+func (mc *basicMirrorClient) getNodeUID() (types.UID, error) {
+	node, err := mc.nodeGetter.Get(mc.nodeName)
+	if err != nil {
+		return "", err
+	}
+	if node.UID == "" {
+		return "", errors.New("node UID unset")
```

Comment:
possibly better to include the nodeName as part of the error message?

Reply:
+1

Reply:
Done.